components/infobox/extensions/wikis/stormgate/infobox_extension_cost_display.lua

```diff
@@ -22,8 +23,7 @@ local ICONS = {
 		default = Abbreviation.make('The', 'Therium'),
 	},
 	buildTime = {
-		--expect to get different ones per faction
-		default = '[[File:Buildtime_terran.gif|baseline|link=Game Speed]]',
+		default = Icon.makeIcon{iconName = 'time', size = '100%'},
```

Comment:
```suggestion
		default = Icon.makeIcon{iconName = 'time', size = 'baseline'},
```
This was how it was before, you sure you want it changed?

Reply:
yeah 100% makes the iicon a bit smaller and more in line with the text around it